identityClient/identityClient.go

```diff
@@ -1297,3 +1299,39 @@ func (c *E3dbIdentityClient) FederatedIdentityKeyCheck(ctx context.Context, para
 	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, request, c.SigningKeys, c.ClientID, nil)
 	return err
 }
+
+// AccessControlPolicy Enables Access Control for an Application Client, returning error (if any).
+func (c *E3dbIdentityClient) AccessControlPolicy(ctx context.Context, params AccessControlPolicyRequest) (*AccessControlPolicyResponse, error) {
```

Comment:
`EnableAccessControlPolicy`, structs can be nouns but methods/functions should be more verby